example3/OpenTelemetry.Metric.Sdk/Aggregators/SumCountMinMax.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Diagnostics.Metric;
+using OpenTelemetry.Metric.Sdk;
+
+namespace OpenTelemetry.Metric.Sdk
+{
+    public class SumCountMinMaxState : AggregatorState
+    {
+        public long count = 0;
+        public double sum = 0;
+        public double max = 0;
+        public double min = 0;
+
+        public override void Update(MeterBase meter, double value)
+        {
+            count++;
+            sum += value;
+            if (count == 1)
+            {
+                min = value;
+                max = value;
+            }
+            else
+            {
+                min = Math.Min(min, value);

```

Comment:
this is just prototype code right and hence not dealing with thread-safety here?

Reply:
correct, there is at least one todo elsewhere noting we aren't thread-safe

Reply:
Agree.  In this particular case, we're safe, as the Collector calls this and that is single instance.

Reply:
you sure its safe? I thought this code was running synchronously on the thread that called Record() and not inside any lock. This means multiple threads could be executing concurrently in this code block.